src/stream/src/task/barrier_manager/progress.rs

```diff
@@ -90,15 +91,24 @@ impl CreateMviewProgress {
     /// Update the progress to `ConsumingUpstream(consumed_epoch)`. The epoch must be monotonically
     /// increasing.
     /// `current_epoch` should be provided to locate the barrier under concurrent checkpoint.
-    pub fn update(&mut self, current_epoch: u64, consumed_epoch: ConsumedEpoch) {
-        match self.state {
-            Some(ChainState::ConsumingUpstream(last)) => {
+    pub fn update(
+        &mut self,
+        current_epoch: u64,
+        consumed_epoch: ConsumedEpoch,
+        rows: ConsumedRows,
+    ) {
+        let consumed_rows = match self.state {
+            Some(ChainState::ConsumingUpstream(last, consumed_row)) => {
                 assert!(last < consumed_epoch);
+                consumed_row + rows
```

Comment:
Seems the `rows` is an accumulated value, and there's no need to sum them up here? 👀

Reply:
For backfill executor, it is not an accumulated value, but make it an accumulated value seems clearer. Let's change it.